sdcm/cluster.py

```diff
@@ -3372,7 +3379,7 @@ def node_setup(self, node, **kwargs):
         # The time will be used in url of Grafana monitor,
         # the data from this point to the end of test will
         # be captured.
-        self.grafana_start_time = time.time()
+        self.grafana_start_time = time.time()   # pylint: disable=attribute-defined-outside-init
```

Comment:
is it possible to define it in the constructor?

Reply:
We can init it to zero, or None